### PR TITLE
Numerical methods for calculate integral in C

### PR DESCRIPTION
**Pull request by OkarinRintaro**
Changes:

Created integral_rectangle.c
Created integral_trapezoid.c

### CODE REVIEW

File: code/numerical_analysis/integral/src/integral_rectangle.c [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    double a, b, step, res;
+    printf("Enter interval: \n");
+    printf("a: \n");
+    scanf("%lf", &a);
+    printf("b: \n");
+    scanf("%lf", &b);
+    printf("Enter integration step: ");
+    scanf("%lf", &step);
+    res = integral_rectangle(a, b, step,
+                             ({
```
Comment (arnavb) [inline code/numerical_analysis/integral/src/integral_rectangle.c]: I don't think you should indent this much, but that is my viewpoint.

Comment (OkarinRintaro) [reply code/numerical_analysis/integral/src/integral_rectangle.c]: I want to clearly distinguish using function in another function. What do you think about it?

**Comment by OkarinRintaro** [comment]
@arnavb , @AdiChat Why my PR not accepted? What something wrong?

File: code/numerical_analysis/integral/src/integral_rectangle.c (hunk 1 above)
Comment (AdiChat) [reply code/numerical_analysis/integral/src/integral_rectangle.c]: No problem 👍

**Comment by AdiChat** [comment]
Everything is fine. Do not worry 😆 
Looks great 😍 

Hope you are enjoying your journey 🚆  with **OpenGenus**

**Pull request merged**